examples/covariance.py

```diff
@@ -0,0 +1,23 @@
+import pyspecdata as psd
+import pyspecProcScripts as psdpr
+
+data = psd.find_file(
+    "241003_27mM_TEMPOL_amp0p1_var_tau_pm_echo",
+    exp_type="ODNP_NMR_comp/Echoes",
+    expno="echo_1",
+    lookup=psdpr.lookup_table,
+)
+data.ift("t2")
+# FID slice
+data = data["t2":(0, None)]
+data *= 2
+data["t2":0] *= 0.5
+# }}}
+# relabel nScans axis with indices
+data.setaxis("nScans", "#")
+with psd.figlist_var() as fl:
+    covariance_matrix = psdpr.select_pathway(
+        data, data.get_prop("coherence_pathway")
+    ).cov_mat("nScans")
+    fl.next("2D Covariance Matrix")
+    fl.image(abs(covariance_matrix))
```

Comment:
I agree very much with making an example that shows the covariance of an FID since we are pretty sure we know what it should look like.  (As opposed to ESR). If it were only for the changes that you made to the function, then I would close the pull request, but I like this. 

Please rename the function so that it's clear we're talking about NMR here, and so that it is named more like the other examples.